test/e2e_node/device_plugin_test.go

```diff
@@ -239,6 +242,16 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 			gomega.Expect(v1ResourcesForOurPod.Containers[0].Devices[0].DeviceIds).To(gomega.HaveLen(1))
 		})
 
+		ginkgo.It("[NodeAlphaFeature:DevicePluginCDIDevices] can make a CDI device accessible in a container", func(ctx context.Context) {
```

Comment:
Could you just enable this feature like here?

https://github.com/kubernetes/kubernetes/blob/a519193da9bb70a92283914f8be2a890c95df7e3/test/e2e_node/node_shutdown_linux_test.go#L95

Reply:
Is there a presubmit you can use to test this job?  I'm not exactly sure where we test device plugins in those jobs.

Reply:
Can it be done only for one test case?

Reply:
After giving this some thought, I'm not sure we need to switch on a feature gate for the test, especially considering an ongoing atempt to graduate this feature to Beta in 1.29. It's better to rely on the default feature gate state instead. WDYT?

Reply:
If the tests work and run without issue, you will have no complaint from me.

Did you find a way to trigger a test of this with presubmits?

Reply:
No, I didn't. @swatisehgal @ffromani any hints?